chromium_src/chrome/browser/first_run/first_run.cc

```diff
@@ -1,19 +1,61 @@
-/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
I'm curious if this was updated due to our lint. If then, was it caused by the "http://mozila..." part in the last line, which is not the "https://", not the year?

Reply:
I think it was just http -> https and updated the year accidentally